cmd/control-plane/Dockerfile

```
@@ -0,0 +1,50 @@
+# syntax=docker/dockerfile:1
+# web builder
+# because this stage builds only web assets, we can use any platform
+FROM --platform=$BUILDPLATFORM node:20.19.0-alpine3.21 AS web
+
+WORKDIR /app
+
+COPY . .
+
+RUN apk add --no-cache make git
+
+RUN make update/web-deps
+RUN make build/web
+
+# pipecd builder
+FROM --platform=$BUILDPLATFORM golang:1.24.1 AS builder
+
+ARG TARGETOS
+ARG TARGETARCH
+
+WORKDIR /app
+
+COPY go.* ./
+RUN go mod download
+
+COPY . ./
+
+RUN make build/go MOD=pipecd BUILD_OS=${TARGETOS} BUILD_ARCH=${TARGETARCH}
```

```suggestion
RUN make build/go MOD=control-plane BUILD_OS=${TARGETOS} BUILD_ARCH=${TARGETARCH}
```
It has to be like this or the build would fail. Please try to run the updated command and build on your local before committing 👀